### PR TITLE
Add icon for parking entrance

### PR DESCRIPTION
**Pull request by dvdmrtnz**
Currently parking entrances have the same icon as the parkings. This PR adds a unique icon for parking entrances.

### CODE REVIEW

**Comment by biodranik** [comment]
It shouldn't be in a blue box, to avoid confusion. You won't drive a car there.

**Comment by dvdmrtnz** [comment]
> It shouldn't be in a blue box, to avoid confusion. You won't drive a car there.

If I’m not mistaken, [amenity=parking_entrance](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dparking_entrance) is used to tag car entrances to underground or multi storey parkings

**Comment by vng** [comment]
Well, funny thing that we already treat parking_entrance as regular parking and draw parking icon there.
The patch have sense, imho.

File: data/styles/clear/include/Icons.mapcss [hunk 1]
```diff
@@ -2673,6 +2673,17 @@ node|z18-[amenity=parking],
 area|z18-[amenity=parking]
 {icon-image: parking-m.svg;font-size: 11;text-optional: true;}
 
+node|z17[amenity=parking_entrance],
+area|z17[amenity=parking_entrance]
+{icon-image: parking_entrance-m.svg;}
+node|z17[amenity=parking_entrance][access=permissive],
+node|z17[amenity=parking_entrance][access=private]
```
Comment (vng) [inline data/styles/clear/include/Icons.mapcss:2680]: "z17-" to void private/permissive icons in all zooms?

**Review by vng — CHANGES_REQUESTED** [review]
I'd prefer merge into one first commit classifier and strings changes.

File: data/styles/clear/include/Icons.mapcss (hunk 1 above)
Comment (dvdmrtnz) [reply data/styles/clear/include/Icons.mapcss:2680]: I copied the same logic that is used just above for amenity=parking

Comment (vng) [reply data/styles/clear/include/Icons.mapcss:2680]: Seems like you forgot to make generate_styles.sh
After that, if you check drules_proto_clear.txt, amenity-parking_entrance-private will be drawable with icon in z18+

Comment (dvdmrtnz) [reply data/styles/clear/include/Icons.mapcss:2680]: Fixed. Thanks!

**Comment by biodranik** [comment]
> If I’m not mistaken, [amenity=parking_entrance](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dparking_entrance) is used to tag car entrances to underground or multi storey parking

OMG you're right ) I confused it with entrance for pedestrians )

File: data/styles/clear/include/Icons.mapcss (hunk 1 above)
Comment (vng) [reply data/styles/clear/include/Icons.mapcss:2680]: Ok, seems like we should revise permissive/private logic here, but in the follow up PRs.

**Review by vng — APPROVED** [review]

**Pull request merged**